### ISSUE TITLE
Unknown layer type: SigmoidCrossEntropyBalanceLoss 

### ISSUE THREAD

**Issue by zhujingsong** [kmaninis/OSVOS-caffe#3]
When I set osvos_params.useTrainOnline = 0 (use the pre-trained models), it works fine. But,
when i set osvos_params.useTrainOnline = 1, there is an error like this

Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: SigmoidCrossEntropyBalanceLoss (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Killed

**Comment by kmaninis** [comment]
Hello, please use the provided Caffe version, where we have made a custom layer called SigmoidCrossEntropyBalanceLoss. The error suggests that in your version of Caffe this layer is not implemented. Please keep us updated for further issues.
